lib/Conversion/TorchToTcp/TcpCustomOpBuilder.cpp

```diff
@@ -0,0 +1,111 @@
+//===------------------------------------------------------------*- C++ -*-===//
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Also available under a BSD-style license. See LICENSE.
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir-tcp/Conversion/TorchToTcp/TcpCustomOpBuilder.h"
+
+#include "torch-mlir/Conversion/Utils/Utils.h"
+#include "torch-mlir/Dialect/Torch/Utils/Utils.h"
+
+namespace mlir::tcp {
+
+void TcpCustomOpBuilder::addOperand(std::string opName, Value value) {
+  // Should there be some error checking here that Value has a specific
+  // type (for instance a RankedTensorType?)
```

Comment:
It should conform to Tcp_Tensor which is defined here: https://github.com/cruise-automation/mlir-tcp/blob/main/include/mlir-tcp/Dialect/IR/TcpTypes.td#L54. But that should automatically happen with the verifier. So, there is no need to explicit error checking here.